index.js

```diff
@@ -56,16 +63,17 @@ load.path = function (dir) {
     typeof __webpack_require__ === 'function' ? 'webpack=true' : '' // eslint-disable-line
   ].filter(Boolean).join(' ')
 
-  throw new Error('No native build was found for ' + target + '\n    loaded from: ' + dir + '\n')
+  throw new Error('No native build was found for ' + target + '\n    loaded from: ' + dir + ' and package: ' + platformPackage + '\n')
```

Comment:
Might make more sense to put `platformPackage` in the `target` array, because it's not relevant when not using platform packages.

Reply:
Are you saying add platformPackage to the target array so the error is:
```
No native build was found for platform=win32 arch=x64 runtime=node abi=83 ... node=16.14.1 package-win32-x64
loaded from: /path/to/library
````
instead of:
````
No native build was found for platform=win32 arch=x64 runtime=node abi=83 ... node=16.14.1
loaded from: /path/to/library and package: library-win32-x64
````
I thought the intent of the error was to list the tags, and then show where node-gyp-build had attempted to resolve/find a binary that matched those tags (and including the platform package name in the list of tags doesn't seem like it would make it clear we were attempting to load from that package).

Reply:
You're right about the intent, and my suggestion doesn't make perfect sense. I just think "loaded from package: library-win32-x64" is equally confusing, in the (atm common) case that a library is _not_ using platform packages - meaning `library-win32-x64` legitimately does not exist. End users could wrongly interpret that as the cause of the error in their logs.

Reply:
Maybe something like this to indicate the primary folder for common/current users:
`tried to load binary from: /path/to/library (and also tried package: library-win32-x64)`
Or I am happy to just take it out, if you prefer.

Reply:
Let's take it out, for now. We can improve the error message later, together with https://github.com/prebuild/node-gyp-build/issues/42. Maybe by that time everyone is using platform packages, who knows :)